src/main/resources/applicationContext.xml

```diff
@@ -81,7 +81,7 @@
 				<prop key="hibernate.format_sql">${hibernate.format_sql:true}</prop>
 				<prop key="hibernate.dialect">${hibernate.dialect:org.hibernate.dialect.MySQLDialect}</prop>
 				<prop key="hibernate.id.new_generator_mappings">false</prop>
-				<prop key="hibernate.hbm2ddl.auto">${hibernate.hbm2ddl.auto:false}</prop>
+				<prop key="hibernate.hbm2ddl.auto">${hibernate.hbm2ddl.auto:none}</prop>
```

Comment:
question: is this correctly handled with the updated pom.xml?

Reply:
Not sure what you mean; `false` wasn't a valid value; but with the newer Hibernate jars, `none` does the same thing.  So I believe it is handled correctly.

Reply:
yes, it's what I mean, thanks!